packages/ui/src/walkthrough-card/WalkthroughCard.css

```diff
@@ -0,0 +1,45 @@
+.ui-WalkthroughCard--CardContainer {
+  display: flex;
+  border-radius: var(--layout-border-radius-xlarge);
+  box-shadow: var(--surface-shadow-medium), var(--surface-shadow-medium);
+  background-color: var(--surface-color-background-white);
+  max-width: var(--walkthrough-card-max-width);
+  flex-direction: column;
+  color: var(--support-border-color-info);
+  padding: var(--layout-padding-large);
+  border: var(--layout-border-width-none);
+  text-align: left;
+  transition: box-shadow var(--transition-medium);
+  text-decoration: none;
+}
+
+.ui-WalkthroughCard--Title {
+  color: var(--support-border-color-info);
+  font: var(--typography-bold-h4);
+  text-decoration: var(--walkthrough-card-title-text-decoration-hidden);
+  transition: text-decoration-color var(--transition-medium);
+  text-underline-offset: var(--walkthrough-card-title-text-underline-offset);
+}
+
+.ui-WalkthroughCard--CardContainer[data-hovered] {
+  cursor: pointer;
+  box-shadow: var(--surface-shadow-large), var(--surface-shadow-large);
+  border: 1px solid var(--surface-color-border-default);
+}
+
```

Comment:
Can we add a data-focus-visible like we do in button and link to make sure our focus outline is the same style as the rest of the site? That way we don't leave it up to the user to determine and get differing outlines?

My focus outline with green highlight color on MacOS
![Screenshot 2024-06-18 at 11 09 04 AM](https://github.com/bcgov/House-Innovations-HOUSNAV/assets/41478918/eec0cead-02a5-4d67-adf7-65404abb9145)